packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -2109,10 +2191,8 @@ describe("AttributesController", function () {
                 }
             });
 
-            await TestUtil.expectThrowsAsync(
-                consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(sharedIdentityAttribute.id),
-                "error.consumption.attributes.invalidPropertyValue"
-            );
+            const result = await consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(sharedIdentityAttribute.id);
```

Comment:
Why should this function no longer throw an error if no RepositoryAttribute ID is used?